sdk/webpubsub/azure-webpubsub-client/azure/webpubsub/client/_model_base.py

```diff
@@ -0,0 +1,705 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
It looks like this has been copied from Core? Any particular reason we're not just using Core?

Reply:
And if there is a particular reason - can we make sure it's documented in comments so that other developers are aware of the context?

Reply:
This `model_base`  keeps consistency with [DPG model](https://github.com/Azure/autorest.python/blob/main/packages/typespec-python/test/generated/core/_specs_/azure/core/_model_base.py). It has not been moved into azure-core so we use it as vendor code